internal/services/pam/pam.go

```diff
@@ -142,15 +142,6 @@ func (s Service) SelectBroker(ctx context.Context, req *authd.SBRequest) (resp *
 		lang = "C"
 	}
 
-	userIsLocked, err := s.userManager.IsUserLocked(username)
```

Comment:
After this change, shouldn't the integration test trace be updated though?

Reply:
Yes. I updated the VHS tape file and the golden files of the SSH integration test now